patches/content-public-browser-render_frame_host.h.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/content/public/browser/render_frame_host.h b/content/public/browser/render_frame_host.h
+index 5fa3f1263137374f433acd4fd6a88b033aa61881..6db6c0fa65bc4e6d0e85c0231fa4f4208226844d 100644
+--- a/content/public/browser/render_frame_host.h
++++ b/content/public/browser/render_frame_host.h
+@@ -124,7 +124,8 @@ class CONTENT_EXPORT RenderFrameHost : public IPC::Listener,
+       const base::android::JavaRef<jobject>& jrender_frame_host_android);
+ #endif
+ 
+-  ~RenderFrameHost() override {}
```

Comment:
I don't understand why this patch is here or the one for the constructor. You don't have to initialize a unique_ptr to null

Reply:
there is a build error otherwise that class with complex members like added `std::unique_ptr` should have constructor and destructor implementations in cc file

Reply:
all patches are removed